backend/exchanges/btcdirect.go

```diff
@@ -17,14 +17,29 @@ package exchanges
 import (
 	"slices"
 
+	"github.com/BitBoxSwiss/bitbox-wallet-app/backend/accounts"
 	"github.com/BitBoxSwiss/bitbox-wallet-app/backend/coins/coin"
 )
 
 const (
 	// BTCDirectName is the name of the exchange, it is unique among all the supported exchanges.
 	BTCDirectName = "btcdirect"
+
+	btcDirectTestApiKey = "6ed4d42bd02eeac1776a6bb54fa3126f779c04d5c228fe5128bb74e89ef61f83"
+
+	btcDirectProdAPiKey = "7d71f633626901d5c4d06d91f7d0db2c15cdf524ddd0ebcd36f4d9c4e04694cd"
+
+	btcDirectUrlTesting = "/btcdirect/fiat-to-coin.html"
```

Comment:
nit: it would be nice to stick to the same naming for URL and apikey
```suggestion
	btcDirectTestUrl = "/btcdirect/fiat-to-coin.html"
```